packages/openapi-to-graphql/src/oas_3_tools.ts

```diff
@@ -280,43 +280,50 @@ function buildUrl(server: ServerObject): string {
 }
 
 /**
- * Returns object | array where all object keys are sanitized. Keys passed in
- * exceptions are not sanitized.
+ * Returns object/array/scalar where all object keys (if applicable) are
+ * sanitized.
  */
-export function sanitizeObjKeys(
-  obj: object | Array<any>,
-  exceptions: string[] = []
-): object | Array<any> {
-  const cleanKeys = (obj: object | Array<any>): object | Array<any> => {
+export function sanitizeObjectKeys(
```

Comment:
We actually have another function called `sanitizeObjectKeys()` in the same file...

I realized that the other `sanitizeObjectKeys()` function was not useful so I ended up removing it but I was surprised to find it.